src/storage/storage_structure/lists/lists.cpp

```diff
@@ -198,12 +201,20 @@ void ListPropertyLists::readFromSmallList(
     diskOverflowFile.readListsToVector(TransactionType::READ_ONLY, *valueVector);
 }
 
+void AdjLists::readFromList(const shared_ptr<ValueVector>& valueVector, ListHandle& listHandle) {
```

Comment:
This code should be refactored in Lists. Maybe a `readListFromPersistentStore` function and then have AdjLists::readFromList call `readListFromPersistentStore`, which Lists::readFromLists calls `readListFromPersistentStore` and then  relUpdateStore->readUpdatesToVectorIfExists(...) (but for write trx only).

Reply:
I moved the call to `readUpdatesToPropertyVectorIfExists()` to readValues:
```
if (transaction->isWriteTransaction()) {
            listsUpdatesStore->readUpdatesToPropertyVectorIfExists(
                storageStructureIDAndFName.storageStructureID.listFileID,
                listHandle.getBoundNodeOffset(), valueVector, listHandle.getStartElemOffset());
        }
```
This function will only be called when we are reading from persistentStore and trx is write.
So we no longer need:
`AdjLists::readFromList(const shared_ptr<ValueVector>& valueVector, ListHandle& listHandle)` function